test/unittests/eof_validation_test.cpp

```diff
@@ -1065,7 +1093,7 @@ TEST(eof_validation, jumpf_into_returning_stack_validation)
     // JUMPF into a function with the same number of outputs as current one
 
     // Exactly required inputs on stack at JUMPF
-    EXPECT_EQ(validate_eof(eof_bytecode(bytecode{OP_STOP})
+    EXPECT_EQ(validate_eof(eof_bytecode(bytecode{OP_CALLF} + "0001" + OP_STOP, 2)
```

Comment:
Confused why this works, this section has `max_stack_height` 0, not 2, doesn't it?